test/support/api/settings.js

```diff
@@ -5,12 +5,25 @@ import fxTestSettings from 'fixtures/test/settings';
 
 const TYPE = 'settings';
 
-Cypress.Commands.add('routeSettings', (mutator = _.identity) => {
-  const data = getResource(fxTestSettings, TYPE);
+Cypress.Commands.add('routeSettings', (setting_key, setting_value) => {
+  let data = getResource(fxTestSettings, TYPE);
+
+  if (setting_key) {
+    const doesSettingExist = _.find(data, { id: setting_key });
+    if (!doesSettingExist) throw new Error(`Setting with the ${ setting_key } key was not found in settings.json fixture.`);
+
+    data = _.map(data, item => {
+      if (item.id === setting_key) {
+        return { ...item, attributes: { value: setting_value } };
+      }
+
+      return item;
+    });
+  }
```

Comment:
This will probably need some refactoring and/or be split off into a separate function. In a "get it working" state.

Reply:
yeah I think just some sort of `buildSettings(fixtures, settingKey, settingValue)` would work.  Is there a reason for the snake_case here?

Then `buildSettings` can just `if (!settingKey) return fixtures;`

Reply:
No particular reason for the snake case, removed it.

I created a `buildSettings()` function and pushed that up 👍